apps/mobile/src/components/ChatDetailScreen/components/MessagesListOrApprovalPreview.tsx

```diff
@@ -24,7 +27,12 @@ export default function MessagesListOrApprovalPreview(): JSX.Element {
       void fetchAndStoreMessagesForInbox({
         key: publicKeyPemBase64,
       })
-    }, [publicKeyPemBase64, fetchAndStoreMessagesForInbox])
+      setTradeChecklistUuid(generateUuid())
```

Comment:
I thought we decided to put this into navigation param, to trigger trading checklist copying data. We can use it in a hook to trigger _chat_ -> _trading checklist_ state sync, while not syncing the navigation when navigating inside tradeChecklistFlow.